app/helpers/fhir_helper.rb

```diff
@@ -241,6 +241,27 @@ def close_contact_from_fhir(related_person)
     }
   end
 
+  def history_as_fhir(history)
+    FHIR::Provenance.new(
+      meta: FHIR::Meta.new(lastUpdated: history.updated_at.strftime('%FT%T%:z')),
+      id: history.id,
+      patient_id: history.patient_id,
+      occurrenceDateTime: history.created_at,
+      recorded: history.updated_at,
+      reason: [
+        {
+          text: history.comment
+        }
+      ],
```

Comment:
My understanding is that this field is meant to represent the "reason that the activity was taking place", so it isn't really a great fit to put `history.comment` here, since `history.comment` really represents what the changes were, not the reason they took place. I think the most honest representation is just to put `history.comment` in a string extension, similar to how we do `exposure-notes` on the Patient.